Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -133,6 +138,16 @@ public struct CustomerCenterConfigData {
                     return "Update"
                 case .updateWarningIgnore:
                     return "Continue"
+                case .platformMismatch:
+                    return "Platform mismatch"
```

Comment:
Happy to talk about this. I went through different ideas, even not showing a title at all. Let me know what you think

![image](https://github.com/user-attachments/assets/2756d0ce-1ea0-43c2-8d1a-cf0828b6dcf4)